images/operator-registry.yml

```diff
@@ -1,9 +1,10 @@
 content:
   source:
+    dockerfile: operator-registry.Dockerfile
```

Comment:
Note: this is needed now as the [upstream operator-registry repository had a root Dockerfile](https://github.com/operator-framework/operator-registry/blob/master/Dockerfile).